### PR TITLE
Some Minor Grammatical Changes

### PR DESCRIPTION
**Pull request by TheoBearman**
Trying to make Cachet look prettier one capital and full stop at a time!

### CODE REVIEW

File: resources/lang/en/cachet.php [hunk 1]
```diff
@@ -88,7 +88,7 @@
     ],
 
     // Other
-    'powered_by'      => ':app Status Page is powered by <a href="https://cachethq.io" class="links">Cachet</a>.',
+    'powered_by'      => ':app Status Page is Powered by <a href="https://cachethq.io" class="links">Cachet</a>.',
```
Comment (jbrooksuk) [inline resources/lang/en/cachet.php]: This doesn't need to be capitalized.

Comment (TheoBearman) [inline resources/lang/en/cachet.php]: Changing back.

File: resources/lang/en/cachet.php [hunk 2]
```diff
@@ -60,7 +60,7 @@
 
     // Subscriber
     'subscriber' => [
-        'subscribe' => 'Subscribe to get the most recent updates.',
+        'subscribe' => 'Subscribe To Get The Most Recent Updates',
```
Comment (GrahamCampbell) [inline resources/lang/en/cachet.php]: surely an uppercase To and The is actually wrong

Comment (TheoBearman) [inline resources/lang/en/cachet.php]: Corrected.

**Comment by jbrooksuk** [comment]
Thanks buddy! :+1:

**Pull request merged**